scripts/setup.sh

```diff
@@ -19,10 +19,9 @@ K8S_VERSION=$1
 export GO111MODULE=on
 
 # setup go module to create the cluster
-go get sigs.k8s.io/kind@v0.5.1
 
 # You can use --image flag to specify the cluster version you want, e.g --image=kindest/node:v1.13.6, the supported version are listed at https://hub.docker.com/r/kindest/node/tags
-kind create cluster --config test/kind-config.yaml --image=kindest/node:$K8S_VERSION
+kind create cluster -v 4 --retain --wait=1m --config test/kind-config.yaml --image=kindest/node:$K8S_VERSION
```

Comment:
> -v 4 --retain --wait=1m

I wonder if these are related to debugging?
Re.`--wait`, do we have to wait for 1 minute? Is it causing any flakes? It seems the default wait time is 0s.

Reply:
`kind` cluster is known to be fast to bring up.
Adding 1 minute wait time for bringing up the cluster slow it down ~10%. Look at https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_kubebuilder/1201/pull-kubebuilder-e2e-k8s-1-16-2/1198845676235001858, the e2e test only takes <10 minutes.

Reply:
I can remove this if desired, but I think it should stay. I matched it to what kind itself does in e2e: https://github.com/kubernetes-sigs/kind/blob/36eae75e067681658d59a31ed679e67e39d9cfc5/hack/ci/e2e-k8s.sh#L88-L93

Without this, kind won't wait at all for the nodes to come up: https://github.com/kubernetes-sigs/kind/blob/6a0de6124de068cd55b24025cf05e8e4ea172641/pkg/internal/cluster/create/actions/waitforready/waitforready.go#L45-L48

If anything, I can reduce `-v` from 4 to 3 I think.

Reply:
It's not adding a static 1 minute wait. It's only checking that the nodes actually come up.

Reply:
note that it's also a fairly tight loop so there is not really additional wait time